### PR TITLE
Add privacy policy to web site

### PR DESCRIPTION
**Pull request by richardgolding**

### CODE REVIEW

File: client/components/public/Privacy.jsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+			your consent, we may use your information that has been anonymized
+			and does not personally identify you. </p>
+		      <p> If you are a resident in the EEA and you
+			believe we are unlawfully processing your
+			personal information, you have the right to
+			complain to your local data protection
+			authority. If you wish to contact us in
+			connection with the exercise of your rights
+			listed above, please email us at&nbsp;
+			<a href="mailto:compliance@greatpuzzlehunt.com">compliance@greatpuzzlehunt.com</a>. We
```
Comment (cabeese) [inline client/components/public/Privacy.jsx:199]: Can you obfuscate the email address? (Is that a thing people still do?) Something like

`<a href='&#109;&#97;ilto&#58;%6&#51;om%7&#48;l&#105;&#97;n%&#54;3%65&#64;gr&#37;65a%74puz%7Al%65%68unt&#46;&#99;&#111;m'>complia&#110;ce&#64;&#103;&#114;&#101;atpuzzlehunt&#46;com</a>`

File: client/components/app/Footer.jsx [hunk 2]
```diff
@@ -7,10 +7,13 @@ export default Footer = class Footer extends Component {
     render() {
     return (
       <div className="ui basic small container segment center aligned">
-        <div className="ui three column grid">
+        <div className="ui four column grid">
           <div className="dark-blue column">
             Great Puzzle Hunt &copy; {eventYear}
           </div>
+          <div className="column">
+	    <a href="/privacy" target="_blank">Privacy Policy</a>
```
Comment (cabeese) [inline client/components/app/Footer.jsx]: I feel like this shouldn't open in a new tab. Can you remove `target="_blank"`?

File: client/components/public/Privacy.jsx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+			  <li> Responding to subpoenas, court orders
+			    or legal process </li>
+			  <li> Enforce our agreements </li>
+			  <li> Protect the health, safety, rights or
+			    property of you, us, or others </li>
+			  <li> Meet legal obligations </li>
+			  <li> Establish or exercise our legal rights
+			    or defend against legal claims </li>
+			  <li> Take action regarding fraud, unlawful
+			    or criminal activity </li>n
```
Comment (cabeese) [inline client/components/public/Privacy.jsx]: erroneous `n` at end of line

**Review by cabeese — COMMENTED** [review]
Looks good - just a couple small things

**Comment by richardgolding** [comment]
I fixed the typo and removed the target on the privacy policy link.

I'm not inclined to obfuscate this address since we don't obfuscate any of the others on the site, and I've not seen anyone do that for some time anyway

**Review by cabeese — APPROVED** [review]
Great, thanks! I'll trust your judgement on email obfuscation :)

**Pull request merged**